### PR TITLE
Add header files to Qhull

### PR DESCRIPTION
**Pull request by asinghvi17**

### CODE REVIEW

**Comment by asinghvi17** [comment]
Should be good to merge!  I've verified that everything seems to have been picked up correctly.

**Comment by giordano** [comment]
Out of curiosity, do you need these files to be explicitly be listed as products because you'll reference them in a Julia package?  Otherwise I'm missing the utility to do it.

File: Q/Qhull/build_tarballs.jl [hunk 1]
```diff
@@ -55,6 +55,20 @@ products = [
     # Libraries
     # reentrant Qhull
     LibraryProduct(["libqhull_r", "qhull_r"], :libqhull_r),
+    
+    # Files
+    # Reentrant Qhull header files
+    FileProduct("include/libqhull_r/geom_r.h", :geom_r),
+    FileProduct("include/libqhull_r/io_r.h", :io_r),
+    FileProduct("include/libqhull_r/libqhull_r.h", :libqhull_r),
```
Comment (giordano) [inline Q/Qhull/build_tarballs.jl:63]: Doesn't this have the same name as the library product above?

Comment (asinghvi17) [reply Q/Qhull/build_tarballs.jl:63]: whoops, it does.  I'll change all of them to `*_r_h`...

**Comment by asinghvi17** [comment]
Yes, we're currently using them in MiniQhull.jl, since wrapping Qhull in Julia is proving to be quite hard.

**Pull request merged**